dev-tools/docker/build_linux_build_image.sh

```diff
@@ -34,7 +34,7 @@ sleep 5
 HOST=docker.elastic.co
 ACCOUNT=ml-dev
 REPOSITORY=ml-linux-build
-VERSION=25
+VERSION=26
```

Comment:
FYI: I've already created a ml-linux-build:26 in order to upgrade to Boost 1.83 in the PR https://github.com/elastic/ml-cpp/pull/2560

Reply:
I think we _could_ override that with the version from this PR if it was ready to merge first.

When it's clearer which PR will be fit to merge first we can decide which one will create the final image 26, and then update the second PR to incorporate the changes of the first and bump the version to 27.